src/hypervisor.c

```diff
@@ -107,6 +108,43 @@ cc_oci_expand_net_device_cmdline(struct cc_oci_config *config, guint index) {
 	return g_strdup("");
 }
 
+/*
+ *
+ *
+ */
+#define DPDK_CMDLINE_OBJ "memory-backend-file,id=dpdkmem,size=2048M,mem-path=/dev/hugepages,share=on,prealloc=on"
```

Comment:
Hey James, I don't think the resizing is particularly likely or necessary for DPDK.  But, we're still just hardcoding a magical value in the code as well as in  the hypervisor.args.  Two hardcoded values which NEED to be the same in two different locations is what gives me angst.